scanner.py

```diff
@@ -280,6 +280,7 @@ def __init__(self, config):
         for sensor in [self.sensor, self.sensor_alt]:
             if sensor:  # Ensure the sensor is not None
                 sensor_name = sensor.upper()
+                self.sensor_name = sensor_name
```

Comment:
Shouldn't we just do this outside the loop?
Doesn't really matter, but just cleaner I think 😄

Reply:
Got a proposal?

Reply:
I think it should just be above line 280.
`self.sensor_name = self.sensor.upper()` :)

Reply:
Needs to make sure `sensor` exists first though?